### DISCUSSION TITLE
got shadowbanned [You are marked as spam, and therefore cannot authorize a third party application.]

### DISCUSSION THREAD

**Discussion by DSDV** [community/community#130888] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, 
a few weeks ago github locked me out of all my messengers and platforms where i used githubs oatuh. 
i only noticed it when i tried to log into jit.si for example. 
no email no notification. Aside from the error message i cant even find any information regarding it in my account. 

The thing is i cant explain why it happened i usually visit a few repos some time s to make suggestions some times to report bugs. 
but lets say not more interaction than once a week or so. 

still the error message reads: You are marked as spam, and therefore cannot authorize a third party application.

lucky i have found alternative ways to contact my friends to say that i am ok. 
but i would like to know what i did to offend github that it shadowbanned me. 

Also why the heck can github without notice block oauth for accounts? 
how am i supposed to contact family and friends? (aside from a new account) 

of course i opened a ticket but i also got ghosted there not even an acknowledgement of my ticket or problem. 

for sure i wont use github for anything anymore. (i hope there will be others to make suggestions and bug reports) 
but i would love a chance to at least use oatuh for a few days to get my affairs in order :( 

so tl:dr if anyone knows how i could get oatuh to work again for a few days it would be greatly appreciated <3
